### PR TITLE
[WIP] NSB101 tutorial content

### PR DESCRIPTION
**Pull request by DavidBoike**
### DO NOT MERGE

Should (hopefully) build and test green, however:

* Solution links are not zipped or really available in any way. Currently result in 404.
* Auto-generated related links on index pages are funky.
* Lessons may need footer to point to our Gitter or other means of support.
* The way lesson pages show up in search results is awkward.

@SimonCropp I'll ping you sometime this week so we can discuss.

### CODE REVIEW

**Comment by indualagarsamy** [comment]
@DavidBoike - when you setup a time, please include me as well.

**Comment by indualagarsamy** [comment]
@DavidBoike @SimonCropp - is this good to go? Seems like there are some build errors.

**Comment by SimonCropp** [comment]
@indualagarsamy no. there are docs engine changes that need to happen to enable it.

File: tutorials/nservicebus-101/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## Table of Contents
+
+1. [Lesson 1: Hello world](lesson-1/)
+2. [Lesson 2: Sending a command](lesson-2/)
+3. [Lesson 3: Managing multiple endpoints](lesson-3/)
+4. [Lesson 4: Publishing events](lesson-4/)
+5. [Lesson 5: Retrying errors](lesson-5/)
+
+
+## Get Started
```
Comment (sergioc) [inline tutorials/nservicebus-101/index.md]: Is this section needed given that you're linking Lesson 1 above?

**Comment by SimonCropp** [comment]
i have started a branch of this. so please hold off pushing any changes

**Comment by DavidBoike** [comment]
Closing this PR as it is replaced by #2114.

Please leave the branch on this PR open for awhile. It's GitHub-readable (no snippets) and there is a short list of external reviewers who have a URL for that branch to review the content before launch.

I'll set a Slack reminder to delete this branch next week.

**Pull request closed without merging**